browser/src/Routes.tsx

```diff
@@ -169,7 +169,9 @@ const Routes = () => {
 
       <Route exact path="/team" component={TeamPage} />
 
-      <Route exact path="/downloads" component={DownloadsPage} />
+      <Redirect from="/downloads" to="/data" />
```

Comment:
I like the redirect to catch old links, however, it appears as though this doesn't allow for old links with a hash to properly link to the new page. e.g. `.../downloads#v4` will just redirect to `.../data`

Not sure if there's an easy way around this. There are a fair number of blog posts that have linked to a specific a tag on the downloads page for ease of user navigation, so just wanted to bring this up.

Reply:
@rileyhgrant nice catch--I am simply going to route both `/downloads` and `/data` to this page.